README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Personal Profile
+## Name: Xumeng Yan
+Aspect | Information
+------------ | -------------
+**Pronoun** | She/Her/Hers
+**Program** | MAPSS
+**Major** | Sociology
+**Research Interest** | *Social Stratification, Social Movements*
+
+* A couple things I enjoy:
+  * Experimenting with cooking recipes
+  * Chasing my pets
+  * Lying on the floor and silently listening to music
+
+* Favorite lyrics:
+> *If you want to make the world a better place*
+> *Take a look at yourself, and then make a change* (Michael Jackson)
+
+![Michael Jackson](Michael_Jackson_-_Man_in_the_Mirror.png)
```

Comment:
It's interesting because when I display this code in Rich Diff, the image does not display. But you have added this binary file above that does show the image. Was this your solution to showing the image?